testsuite/fuzzer/google-oss-fuzz/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://aptos.dev"
+language: rust
+primary_contact: "gerardo@aptoslabs.com"
+main_repo: "https://github.com/aptos-labs/aptos-core"
+auto_ccs:
+  - "davidiw@aptoslabs.com"
+  - "security@aptoslabs.com"
+  - "wg@aptoslabs.com"
```

Comment:
@wrwg @davidiw these lines grant you access to Google's ClusterFuzz and notifies you when new issues are created